.github/workflows/native-image-logging-examples.yml

```diff
@@ -0,0 +1,40 @@
+name: native-image-logging-examples
+on:
+  push:
+    paths:
+      - 'native-image-logging-examples/**'
+      - '.github/workflows/native-image-logging-examples.yml'
+  pull_request:
+    paths:
+      - 'native-image-logging-examples/**'
+      - '.github/workflows/native-image-logging-examples.yml'
+  schedule:
+    - cron: "0 0 1 * *" # run every month
+  workflow_dispatch:
+permissions:
+  contents: read
+  pull-requests: write # for `native-image-pr-reports` option
+jobs:
+  run:
+    name: Run 'native-image-logging-examples'
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v3
+      - uses: graalvm/setup-graalvm@v1
+        with:
+          version: 'dev'
+          java-version: '17'
+          components: 'native-image'
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          native-image-job-reports: 'true'
+          native-image-pr-reports: 'true'
```

Comment:
Note that this only works for the last native image run, which builds the `RuntimeLoggerInit` demo. The data for the previous build is overwritten, but I think that may be ok.